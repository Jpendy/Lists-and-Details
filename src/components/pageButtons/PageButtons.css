.buttonSection {
    display: flex;
    justify-content: center;
}

.buttonSection button {
    width: 80px;
    height: 40px;
    margin: 20px;
    border-radius: 20px;
    box-shadow: 1px 2px 5px 1px;
    cursor: pointer;
    background: yellow;
    transition: .02s;
    &:hover {
        background: lime;
    }
}