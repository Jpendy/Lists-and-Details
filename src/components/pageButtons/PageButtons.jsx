import React from 'react';
import PropTypes from 'prop-types';

const PageButtons = ({ page, handleClick }) => (
  <>
    {page < 25 && <button name='next' onClick={handleClick}>Next</button>}
    {page > 1 && <button name='previous' onClick={handleClick} >Previous</button>}
  </>
);

PageButtons.propTypes = {
  page: PropTypes.number.isRequired,
  handleClick: PropTypes.func.isRequired
};

export default PageButtons;

