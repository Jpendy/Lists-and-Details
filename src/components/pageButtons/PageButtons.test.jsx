import React from 'react';
import { shallow } from 'enzyme';
import PageButtons from './PageButtons';

describe('PageButtons component', () => {
  it('renders PageButtons', () => {
    const wrapper = shallow(<PageButtons page={1} handleClick={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
  
