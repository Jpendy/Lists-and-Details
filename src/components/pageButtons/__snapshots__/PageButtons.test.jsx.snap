// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageButtons component renders PageButtons 1`] = `
<Fragment>
  <button
    name="next"
    onClick={[Function]}
  >
    Next
  </button>
</Fragment>
`;
